CONTRIBUTING.md

```diff
@@ -1,12 +1,14 @@
 # Contributing notes
 
-pmm-managed is highly integrated with PMM Server environment. As a result, development and testing is done partially inside PMM Server container which we call "devcontainer".
+pmm-managed is highly integrated with PMM Server environment. As a result, development and testing is done partially inside PMM Server container which we call "devcontainer". You can take a look at the architecture [here](https://www.percona.com/doc/percona-monitoring-and-management/2.x/details/architecture.html)
 
 
 # Devcontainer setup
 
 Install Docker and Docker Compose.
 
+Checkout `PMM-2.0` branch which is the main branch for PMM 2.x development.
+
 Run `make` to see a list of targets that can be run on host:
 ```
 $ make
```

Comment:
shoffman@darkhorse ~/Documents/GitHub/pmm $ make
make: *** No targets specified and no makefile found.  Stop.

Reply:
shoffman@darkhorse ~/Documents/GitHub/pmm $ git status
On branch PMM-2.0
Your branch is up to date with 'origin/PMM-2.0'.

nothing to commit, working tree clean

Reply:
> shoffman@darkhorse ~/Documents/GitHub/pmm

are you in the correct directory? the path should end with pmm-managed

i think you ran it from the pmm repo (the API repo) which has its Makefile in pmm/api directory  https://github.com/percona/pmm/tree/PMM-2.0/api

Reply:
Yep, brain fart!